projects/1.20/assets/bygone-nether/bygonenether/lang/zh_cn.json

```
@@ -0,0 +1,111 @@
+	"item.bygonenether.warped_nether_brick_vertical_slab": "诡异下界砖竖直台阶",
+
+	"block.bygonenether.cobbled_blackstone": "黑石圆石",
+	"block.bygonenether.withered_blackstone": "凋灵黑石",
+	"block.bygonenether.withered_blackstone_stairs": "凋灵黑石楼梯",
+	"block.bygonenether.withered_blackstone_slab": "凋灵黑石台阶",
+	"block.bygonenether.chiseled_withered_blackstone": "雕纹凋灵黑石",
+	"block.bygonenether.cracked_withered_blackstone": "裂纹凋灵黑石",
+	"block.bygonenether.cracked_withered_blackstone_stairs": "裂纹凋灵黑石楼梯",
+	"block.bygonenether.cracked_withered_blackstone_slab": "裂纹凋灵黑石台阶",
+	"block.bygonenether.withered_debris": "凋灵残骸",
+	"block.bygonenether.soul_stone": "灵魂石",
+	"block.bygonenether.warped_nether_bricks": "诡异下界砖块",
+	"block.bygonenether.chiseled_warped_nether_bricks": "雕纹诡异下界砖",
+	"block.bygonenether.warped_nether_brick_stairs": "诡异下界砖楼梯",
+	"block.bygonenether.warped_nether_brick_slab": "诡异下界砖台阶",
+	"block.bygonenether.withered_basalt": "凋灵玄武岩",
+	"block.bygonenether.withered_coal_block": "凋灵煤炭块",
+	"block.bygonenether.withered_quartz_block": "凋灵石英块",
+	"block.bygonenether.netherite_bell": "下界合金钟",
+
+	"block.bygonenether.withered_blackstone_vertical_slab": "凋灵黑石竖直台阶",
+	"block.bygonenether.cracked_withered_blackstone_vertical_slab": "裂纹凋灵黑石竖直台阶",
+	"block.bygonenether.warped_nether_brick_vertical_slab": "诡异下界砖竖直台阶",
+
+	"entity.bygonenether.piglin_prisoner": "猪灵囚犯",
+	"entity.bygonenether.piglin_hunter": "猪灵猎人",
+	"entity.bygonenether.wex": "凋灵恼鬼",
+	"entity.bygonenether.warped_enderman": "诡异末影人",
+	"entity.bygonenether.wraither": "凋灵死士",
+	"entity.bygonenether.corpor": "凋灵蛮兵",
+	"entity.bygonenether.wither_skeleton_knight": "凋灵骷髅骑士",
+	"entity.bygonenether.wither_skeleton_horse": "凋灵骷髅马",
+
+	"subtitle.bygonenether.entity.warped_enderman.ambient": "诡异末影人：低鸣",
+	"subtitle.bygonenether.entity.warped_enderman.death": "诡异末影人：死亡",
+	"subtitle.bygonenether.entity.warped_enderman.hurt": "诡异末影人：受伤",
+	"subtitle.bygonenether.entity.warped_enderman.scream": "诡异末影人：尖叫",
+	"subtitle.bygonenether.entity.warped_enderman.stare": "诡异末影人：喊叫",
+	"subtitle.bygonenether.entity.warped_enderman.teleport": "诡异末影人：传送",
+	"subtitle.bygonenether.entity.wex.ambient": "凋灵恼鬼：恼人",
+	"subtitle.bygonenether.entity.wex.death": "凋灵恼鬼：死亡",
+	"subtitle.bygonenether.entity.wex.hurt": "凋灵恼鬼：受伤",
+	"subtitle.bygonenether.entity.wex.charge": "凋灵恼鬼：尖叫",
+
+	"advancements.bygonenether.acquire_wither_waltz.title": "跳一支华尔兹",
+	"advancements.bygonenether.acquire_wither_waltz.description": "获得《Wither Waltz》音乐唱片",
+	"advancements.bygonenether.catacomb.title": "生存还是凋零……",
+	"advancements.bygonenether.catacomb.description": "找到一处地下墓窟",
+	"advancements.bygonenether.citadel.title": "中心堡垒",
+	"advancements.bygonenether.citadel.description": "找到一处下界堡垒",
+	"advancements.bygonenether.explore_structures.title": "更热门的景点",
+	"advancements.bygonenether.explore_structures.description": "到访本模组内所有建筑",
+	"advancements.bygonenether.piglin_manor.title": "小心身后",

```

```suggestion
	"advancements.bygonenether.piglin_manor.title": "小心追击",
```
找了这东西的出典，仅建议，我也不知道怎么把“庄园”这个概念放进去